lib/doofinder_api_items.php

```diff
@@ -70,4 +93,14 @@ private function post($url, $payload)
 
         return json_decode($response->response, true);
     }
+
+    /**
+     * Get Process Callback URL
+     *
+     * @return string
+     */
+    private function getProcessCallbackUrl()
+    {
+        return Context::getContext()->link->getModuleLink('doofinder', 'callback', []);
+    }
```

Comment:
also, out

Reply:
Done on https://github.com/doofinder/doofinder-prestashop/pull/143/commits/13a4e4631a3fff59967320600ca685099550cd59. Thank you for the feedback! :smile: